sysdig/resource_sysdig_secure_cloud_account_v2.go

```diff
@@ -0,0 +1,96 @@
+package sysdig
+
+import (
+	"context"
+	"time"
+
+	v2 "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceSysdigSecureCloudAccountV2() *schema.Resource {
```

Comment:
Similar to my comment on Ravina's PR, would it make more sense to name this `SysdigSecureCloudauthAccount` as opposed to `SysdigSecureCloudAccountV2`? Peppering versions into variable/function names feels a little strange

Reply:
Update the changes in this PR: https://github.com/sysdiglabs/terraform-provider-sysdig/pull/407.

Closing this draft PR.